conda-forge.yml

```diff
@@ -10,5 +10,4 @@ build_platform:
     linux_aarch64: linux_aarch64  # ARM for Linux
     win_64: win_64
     osx_64: osx_64
-    osx_arm64: osx_64
-test: native_and_emulated
```

Comment:
Back then a native osx_64 was provided, it was required to run oxs_arm64 running on osx_64 but we now have native support for `osx_arm64`